src/Semaphore/SemaphoreCharacter.css

```diff
@@ -7,41 +7,41 @@
 }
 
 .Semaphore .SemaphoreCheckbox.SemaphoreCharacter-north {
```

Comment:
You want to chance all usages of the `.Semaphore` class to be the `.SemaphoreCharacter` class.  I added a new layer without updating the CSS file.